x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -199,6 +200,9 @@ export const ENABLE_ASSET_CRITICALITY_SETTING = 'securitySolution:enableAssetCri
 export const EXCLUDED_DATA_TIERS_FOR_RULE_EXECUTION =
   'securitySolution:excludedDataTiersForRuleExecution' as const;
 
+/** This Kibana Advances setting allows users to define the maximum amount of unassociated notes (notes without a `timelineId`) */
+export const MAX_UNASSOCIATED_NOTES = 'securitySolution:maxUnassociatedNotes' as const;
```

Comment:
Consider renaming this to something as below because current name feels like it is the number of MAX_NOTES but not the setting name.

```suggestion
export const MAX_UNASSOCIATED_NOTES_SETTING = 'securitySolution:maxUnassociatedNotes' as const;
```